mintapi/api.py

```diff
@@ -263,8 +266,6 @@ def get_categories(self):  # {{{
         return categories
 
     def get_budgets(self):  # {{{
-        # Get categories
-        categories = self.get_categories()
```

Comment:
I think it would be best to keep this here, so we can pass it to get_category_from_id, otherwise it is calling self.get_categories() quite a few times for no particular reason. Even if it happens to be a no-op after the first call, that's still an implementation detail that we shouldn't assume :)